docs/geps/gep-04.rst

```diff
@@ -0,0 +1,303 @@
+
+We implemented a small set of additional features that simplify the specification of
+certain types of functions of the tax and transfer system.
+
+
+.. _gep-4-group-sums:
+
+Group summation of variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Individual-level measures often need to be summed up to a group level. GETTSIM automates
+this process for groups of individuals defined in :ref:`gep-1` (households ``_hh`` and
+tax units ``_tu``).
+
+In case an individual-level column `[column]` exists, the graph will be augmented with a
+node including a group sum like `[column]_hh` should that be requested. Requests can be
+either inputs in a downstream function or explicit targets of the calculation.
+
+Automatic summation will only happen in case no column `[column]_hh` is explicitly set.
+Using a different reduction function than the sum is as easy as explicitly specifying
+`[column]_hh`.
+
+Consider the following example: the function ``kindergeld_m`` calculates the
+individual-level child benefit payment. ``arbeitsl_geld_2_m_hh`` calculates
+Arbeitslosengeld 2 on the household level (as indicated by the suffix). One necessary
+input of this function is the sum of all child benefits on the household level. There is
+no function or input column ``kindergeld_m_hh``.
+
+By including ``kindergeld_m_hh`` as an argument in the definition of
+``arbeitsl_geld_2_m_hh`` as follows:
+
+.. code-block:: python
+
+    def arbeitsl_geld_2_m_hh(
+        kindergeld_m_hh: FloatSeries, other_arguments: FloatSeries
+    ) -> FloatSeries:
+        ...
+
+a node ``kindergeld_m_hh`` containing the household-level sum of ``kindergeld_m`` will
+be automatically added to the graph. Its parents in the graph will be ``kindergeld_m``
+and ``hh_id``.
+
+
+.. _gep-4-time-unit-conversion:
+
+Conversion between reference periods
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Similarly to :ref:`gep-4-group-sums`, GETTSIM will automatically convert values
+referring to different reference periods defined in :ref:`gep-1` (years (default, no
+suffix), months ``_m``, and days ``_t``).
+
+In case a column with annual values `[column]` exists, the graph will be augmented with
+a node including monthly values like `[column]_m` should that be requested. Requests can
+be either inputs in a downstream function or explicit targets of the calculation. In
+case the column refers to a different level of aggregation, say ``[column]_hh``, the
+same applies to ``[column]_m_hh``.
+
+Automatic summation will only happen in case no column `[column]_m` is explicitly set.
+Using a different conversion function than the sum is as easy as explicitly specifying
+`[column]_m`.
+
+Conversion goes both ways and uses the following formulas:
+
++-----------+--------+---------+
+| time unit | suffix | factor  |
++-----------+--------+---------+
+| year      |        | 1       |
++-----------+--------+---------+
+| month     | `_m`   | 12      |
++-----------+--------+---------+
+| week      | `_w`   | 52.1775 |
```

Comment:
I get 365.25 * 7 ~ 52.1785. Should be consistent between week and day

Reply:
Excellent, thanks for checking! I confess I just used google... Fixed in f97ce.